src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -1654,13 +1655,13 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
      * 
      */
 
-    protected void updateProblem() {
+    protected boolean updateProblem() {
 
         // showStackTrace();
 
         if (!validateProblemFields()) {
             // problem defined by the GUI fields is invalid, just return ( error message was issued by validateProblemFields() )
```

Comment:
Change comment to indicate `false `is returned since it no longer just "returns".